server/l10n/en/LC_MESSAGES/messages.po

```diff
@@ -0,0 +1,127 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#. The label for a collection of exploration tools.
+#: server/templates/base.html:69 server/templates/base.html:120
+msgid "Explore"
+msgstr ""
+
```

Comment:
For my education, is msgstr used for translator to get a better idea about the string?

Reply:
technically, the msgstr should contain the string, whereas the id is a key. the way jinja-babel is set up, there isn't a way to specify the key, and the id is used in the default language.

my understanding of the file format is more like this, with separate id and str: https://github.com/datacommonsorg/website/blob/i18n/server/l10n/chart_titles.pot